app/components/space-setting-modal/template.hbs

```diff
@@ -21,14 +21,11 @@
     {{input value=space.server.username placeholder="username"}}
   </p>
   <p>
-    {{input value=space.server.password placeholder="password"}}
+    {{input type="password" value=space.server.password placeholder="password"}}
   </p>
   <p>
     {{input value=space.server.nickname required="required" placeholder="nickname"}}
   </p>
-  <p>
-    username:  {{space.server.username}}
-  </p>
```

Comment:
This is removing the username?

Reply:
No, this was outputting the username below the form from the actual username input field. Don't know why this was there in the first place, I guess it was for debugging purposes.

Reply:
Ah, ok.